src/signalrclient/logger.cpp

```diff
@@ -18,8 +26,16 @@ namespace signalr
         {
             try
             {
+                time_t t;
```

Comment:
nit: maybe we can clean this up a bit? I have no idea about all of these types.

Reply:
Cleaned it up a little